Rocket.Chat/Models/Message/UnmanagedMessage.swift

```diff
@@ -177,13 +190,141 @@ extension UnmanagedMessage {
     func isSystemMessage() -> Bool {
         return !(
             type == .text ||
-            type == .audio ||
-            type == .image ||
-            type == .video ||
-            type == .textAttachment ||
-            type == .url
+                type == .audio ||
+                type == .image ||
+                type == .video ||
+                type == .textAttachment ||
+                type == .url
         )
     }
+
+    // swiftlint:disable function_body_length cyclomatic_complexity
```

Comment:
This methods are being duplicated for unmanaged objects... I believe we can remove them from the managed objects in the future.